### PR TITLE
Updated the hyperlink

### PR DESCRIPTION
**Pull request by ananthulasrikar**
### Describe the proposed changes

URL (which is text)is updated with hyperlink

### Please take a look

https://docs.docker.com/engine/installation/ubuntulinux/ is replaced with a href tag so that its visible as link & open in new tab

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview ready!

Built with commit d297602d8ad03561675bf19a566101fc12073dfe

https://deploy-preview-668--docker-docs.netlify.com

File: engine/installation/cloud/cloud-ex-aws.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ To get help with your private key file, instance IP address, and how to log into
         $ cd ~/Desktop/keys/amazon_ec2
         $ ssh -i "my-key-pair.pem" ubuntu@xx.xxx.xxx.xxx
 
-    We'll follow the instructions for installing Docker on Ubuntu at https://docs.docker.com/engine/installation/ubuntulinux/. The next few steps reflect those instructions.
+    We'll follow the instructions for installing Docker on Ubuntu at <a href="https://docs.docker.com/engine/installation/ubuntulinux/" target="_blank">https://docs.docker.com/engine/installation/ubuntulinux/</a>. The next few steps reflect those instructions.
```
Comment (johndmulhausen) [inline engine/installation/cloud/cloud-ex-aws.md]: Can you do this link in markdown instead of HTML please? Thanks!

Comment (ananthulasrikar) [inline engine/installation/cloud/cloud-ex-aws.md]: Thanks for the inputs @johndmulhausen . I have updated and sent a new pull request. Hence we can close this.

**Comment by ananthulasrikar** [comment]
Sent a new [pull request](https://github.com/docker/docker.github.io/pull/680)

**Pull request closed without merging**